### DISCUSSION TITLE
[mini.completion] How to scroll docs popup?

### DISCUSSION THREAD

**Discussion by GitMurf** [nvim-mini/mini.nvim#1260] [Q&A] [answered]
I didn't see an option in the config mappings and the usual `<C-d/u>` or `<C-f/b>` was not working. How do we scroll the docs popup? Thanks!

**Comment by echasnovski** [comment, marked as answer]
I am afraid there is no way. Probably never will be due to how most of the `<C-*>` keys are populated in Insert mode.

**Reply by GitMurf** [reply]
Ok, but just realized that we can somewhat get around this by setting the window options in mini.completion config. For example:

```lua
    window = {
      info = { height = 75, width = 80, border = 'double' },
      -- ... other stuff
    },
```

Any issue with this @echasnovski ? it seems like it works nicely where the height is just the max, so when smaller it will shrink / size accordingly. This just allows for ability to maximize the potential size of the window on large documentation items.

**Reply by echasnovski** [reply]
If this works for you, then great. I personally prefer not having completion popup and windows span across the whole height.
You can also set `height = vim.o.lines` (or `height = vim.o.lines - 3` to account for tabline+statusline+cmdline) and it may increase up to the whole terminal instance height.
